### DISCUSSION TITLE
Folder Download

### DISCUSSION THREAD

**Discussion by trippletrouble** [developeranaz/Mega.nz-Rclone-Index-Heroku#22] [Q&A]
Is there any way to download it as a folder?

**Comment by developeranaz** [comment]
> Is there any way to download it as a folder?

Foldered download can be possible by using Network Webdav this dude had made that 
https://github.com/bluehypergiant/Mega.nz-rclone-WebDav 

or other alternative method is by using any scrapers or link grabbers (folder inside folder is not possible using this method ) https://github.com/developeranaz/rc-web-scraper

**Reply by trippletrouble** [reply]
Thank you so much. Your app saved my life. Will surely donate soon !

**Reply by trippletrouble** [reply]
I tried but I tried the first link but I dont know how to download as folder.

**Reply by developeranaz** [reply]
> I tried but I tried the first link but I dont know how to download as folder.

I will make a short video on YouTube, everyone is asking for it. Now I will explain it

If you are using windows
 - by using webdav 
## How to Connect to webdav ?
go to thispc > rightclick on thispc >add network location
![Screenshot_20211028_123814](https://user-images.githubusercontent.com/71500526/139204660-07b1e31d-09e0-4a51-a04c-f371352c70dd.jpg)

**Reply by developeranaz** [reply]
i didn't ask you, what are you trying to download is it `folder inside folders` or `just folder inside files`

**Reply by trippletrouble** [reply]
@developeranaz folder inside folders

**Reply by developeranaz** [reply]
> > I tried but I tried the first link but I dont know how to download as folder.
> 
> I will make a short video on YouTube, everyone is asking for it. Now I will explain it
> 
> If you are using windows
> 
> * by using webdav
> 
> ## How to Connect to webdav ?
> go to thispc > rightclick on thispc >add network location ![Screenshot_20211028_123814](https://user-images.githubusercontent.com/71500526/139204660-07b1e31d-09e0-4a51-a04c-f371352c70dd.jpg)

then this is the only way

**Reply by trippletrouble** [reply]
@developeranaz what should I do after add a network location

**Reply by developeranaz** [reply]
> @developeranaz what should I do after add a network location

after adding your mega credentials, a new network drive is displayed in your `thispc` 
just copy and paste your folders like normal copying stuffs.

There is no other way, maybe in soon or later i'll make a scraper for both folder inside folders way.
